Core/Relationships/RelationshipResolver.cs

```diff
@@ -333,6 +334,15 @@ private void Add(CkanModule module, Relationship reason)
             }
         }
 
+        private bool MightBeInstallable(CkanModule module)
```

Comment:
- [ ] Some docs on this would be great.

It looks like we recursively chase all dependencies and make sure we can potentially install them all in some iteration. (Although we don't check against conflicts against currently installed mods, since `MightBeInstallable` vs `AbsolutelyInstallable`. :)